monkey/tests/unit_tests/infection_monkey/ransomware/test_ransomware_payload.py

```diff
@@ -51,15 +34,46 @@ def ransomware_payload(build_ransomware_payload, ransomware_payload_config):
 
 
 @pytest.fixture
-def build_ransomware_payload(telemetry_messenger_spy):
+def build_ransomware_payload(
```

Comment:
This whole file looks like a "mockery". What's the point of unit tests if we only assert that something got called? Aren't these kind of tests just tying everything together and making it harder to refactor, without actually testing any logic?

Reply:
It does test some logic. For example, it tests that encryption and readme actions are only performed if they are enabled in the configuration. It also tests that these actions are only performed if a target directory was specified. In addition, it tests that telemetries are constructed with the correct information.

There are more examples, but those are just a few.